test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/Binders/ArrayModelBinderTest.cs

```diff
@@ -44,23 +44,23 @@ public class ArrayModelBinderTest
             // Arrange
             var binder = new ArrayModelBinder<string>(new SimpleTypeModelBinder(typeof(string)));
 
-            var context = CreateContext();
-            context.IsTopLevelObject = true;
+            var bindingContext = CreateContext();
```

Comment:
It was easier to just rename + find and replace. Tests often used different naming within the same file and this makes it a little bit better.